integrations/api_repo_git_notes_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package integrations
+
+import (
+	"net/http"
+	"net/url"
+	"testing"
+
+	"code.gitea.io/gitea/models"
+	api "code.gitea.io/gitea/modules/structs"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAPIReposGitNotes(t *testing.T) {
+	onGiteaRun(t, func(*testing.T, *url.URL) {
+		user := models.AssertExistsAndLoadBean(t, &models.User{ID: 2}).(*models.User)
+		// Login as User2.
+		session := loginUser(t, user.Name)
+		token := getTokenForLoggedInUser(t, session)
+
+		// check invalid requests
+		req := NewRequestf(t, "GET", "/api/v1/repos/%s/repo1/git/notes/12345?token="+token, user.Name)
+		session.MakeRequest(t, req, http.StatusNotFound)
+
+		req = NewRequestf(t, "GET", "/api/v1/repos/%s/repo1/git/notes/..?token="+token, user.Name)
+		session.MakeRequest(t, req, http.StatusUnprocessableEntity)
+
+		// check valid request
+		req = NewRequestf(t, "GET", "/api/v1/repos/%s/repo1/git/notes/65f1bf27bc3bf70f64657658635e66094edbcb4d?token="+token, user.Name)
+		resp := session.MakeRequest(t, req, http.StatusOK)
```

Comment:
Mixing two styles? Maybe this:

```go
req: = NewRequestf(t, "GET", "/api/v1/repos/%s/repo1/git/notes/12345?token=%s", token, user.Name)
```

should work as well? If so, it looks more consistent.

Likewise for the other two similar instances.

Reply:
Agreed. It is indeed more consistent.